.github/workflows/backport-reminder.yml

```diff
@@ -0,0 +1,24 @@
+# This workflow sends a reminder comment to PRs that have labels starting with
+# `backport/` to check that the backport has run successfully.
+
+on:
+  pull_request:
+    types: [ labeled ]
+    # Runs on PRs to main and all release branches
+    branches:
+      - main
+      - release/*
+
+jobs:
+  backport-label-check:
+    if: "startsWith(github.event.label.name, 'backport/')"
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Comment on PR
+        run: |
+          github_message="After merging, confirm that you see messages like: 🍒✅ Cherry pick of commit ... onto ... succeeded!"
+          curl -s -H "Authorization: token ${{ secrets.PR_COMMENT_TOKEN }}" \
```

Comment:
Curious, does the default `GH_TOKEN` not have permissions to do this?

Reply:
To be honest, I just lifted this from another workflow 😆 Not sure!